_stbt/core.py

```diff
@@ -689,6 +689,18 @@ def __repr__(self):
         return str(self)
 
 
+class OcrPatterns(object):
+    """
+    Common tesseract_user_patterns arguments you might want to pass when calling
+    ocr.
+
+    >>> import stbt
+    >>> stbt.ocr(frame=load_image("tests/ip_address.png"))
+    u'192.168.1.64'
```

Comment:
The doctest doesn't use OcrPatterns / tesseract_user_patterns!

Reply:
The doctest code is potentially confusing to people who aren't expecting to pass in a frame explicitly? Might be better suited as a test in `tests/test_ocr.py`.